Tasks/DownloadBuildArtifactsV0/download_handlers/handlers_implementation.ts

```diff
@@ -0,0 +1,190 @@
+import { IBaseHandlerConfig, IContainerHandlerConfig, IContainerHandlerZipConfig } from './handlers_config'
+import { handlerCheckDownloadedFiles } from "../download_helper";
+import { WebProvider, FilesystemProvider, ZipProvider, } from 'artifact-engine/Providers';
+import { ArtifactEngine } from 'artifact-engine/Engine';
+import { IArtifactProvider, ArtifactDownloadTicket } from 'artifact-engine/Models';
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as DecompressZip from "decompress-zip";
+
+abstract class DownloadHandler {
+    config: IBaseHandlerConfig;
```

Comment:
Can we make this protected - for the sake of encapsulation? 😃

Reply:
Fixed it, thanks!